pkg/models/plan_discussion.go

```diff
@@ -8,7 +8,7 @@ import (
 type PlanDiscussion struct {
 	baseStruct
 	modelPlanRelation
-	Content             string              `json:"content" db:"content"`
+	Content             TaggedHTMLInput     `json:"content" db:"content"`
```

Comment:
This is set as TaggedHTMLInput instead of TaggedHTML, so that we can return a type in GQL instead of a scalar